tests/integ/test_udf.py

```diff
@@ -409,10 +408,8 @@ def test_session_register_udf(session):
     )
 
 
-@pytest.mark.skipif(
-    not is_pandas_available, reason="pandas is required to register vectorized UDFs"
-)
```

Comment:
I don't understand why this is being removed

Reply:
This was only necessary because the original test contains a pandas UDF registration. I have split the test into 5 tests below, `test_register_vectorized_udf_from_file` has the `skipif` makrer.